package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@cognite/react-timeseries-chart",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

```suggestion
  "version": "0.2.0",
```
If you follow semantic versioning :)